### PR TITLE
version=1.1.28

### PR DESCRIPTION
**Pull request by Asit-fovus**

### CODE REVIEW

File: cli_arguments.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <!DOCTYPE html>
-<html class="writer-html5" lang="en" >
+<html class="writer-html5" lang="en" data-content_root="./">
 <head>
-  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />
+  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (xuanyi-chen) [inline cli_arguments.html:4]: Why?

File: cli_arguments.html [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Fovus CLI &mdash; Fovus Documentation 1.0.0 documentation</title>
-      <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
-      <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
+      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=92fd9be5" />
+      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=19f00094" />
```
Comment (xuanyi-chen) [inline cli_arguments.html:9]: Why?

**Review by xuanyi-chen — APPROVED** [review]

File: cli_arguments.html (hunk 1 above)
Comment (Asit-fovus) [reply cli_arguments.html:4]: auto-generated code

File: cli_arguments.html (hunk 2 above)
Comment (Asit-fovus) [reply cli_arguments.html:9]: auto-generated code

Comment (xuanyi-chen) [reply cli_arguments.html:9]: your change does not look right

File: cli_arguments.html (hunk 1 above)
Comment (xuanyi-chen) [reply cli_arguments.html:4]: your change does not look right
try to install new libs

Comment (Asit-fovus) [reply cli_arguments.html]: tried to uninstall and reinstall sphinx-related libs. but still, the same files are generated. for now, I have manually discarded those changes

File: cli_arguments.html (hunk 2 above)
Comment (Asit-fovus) [reply cli_arguments.html]: tried to uninstall and reinstall sphinx-related libs. but still, the same files are generated. for now, I have manually discarded those changes

**Review by xuanyi-chen — APPROVED** [review]

**Review by xuanyi-chen — APPROVED** [review]

**Pull request merged**